pallets/services/src/lib.rs

```diff
@@ -1673,5 +1622,219 @@ pub mod module {
+			security_commitments: Vec<AssetSecurityCommitment<T::AssetId>>,
+		) -> DispatchResultWithPostInfo {
+			let caller = ensure_signed(origin)?;
+
+			// Ensure all operators are active
+			for operator in operators.iter() {
+				ensure!(
+					T::OperatorDelegationManager::is_operator_active(operator),
+					Error::<T>::OperatorNotActive,
+				);
+			}
+
+			// Ensure each asset has non-zero exposure requirements
+			for requirement in asset_security_requirements.iter() {
+				ensure!(
+					requirement.min_exposure_percent > Percent::zero()
+						&& requirement.max_exposure_percent > Percent::zero()
+						&& requirement.min_exposure_percent <= requirement.max_exposure_percent
+						&& requirement.max_exposure_percent <= Percent::from_percent(100),
+					Error::<T>::InvalidSecurityRequirements,
+				);
+			}
+
+			// Ensure no duplicate operators
+			let mut seen_operators = BTreeMap::new();
+			for operator in operators.iter() {
+				ensure!(!seen_operators.contains_key(operator), Error::<T>::DuplicateOperator);
+				seen_operators.insert(operator.clone(), ());
+			}
+
+			let (_, _blueprint) = Self::blueprints(blueprint_id)?;
+
+			// Ensure all operators are registered for this blueprint
+			for operator in operators.iter() {
+				ensure!(
+					Operators::<T>::contains_key(blueprint_id, operator),
+					Error::<T>::NotRegistered
+				);
+			}
+
+			// Verify that we have a signature from each operator
+			let mut operator_signatures = BTreeMap::new();
+			for (operator, signature) in quote_signatures.iter() {
+				ensure!(operators.contains(operator), Error::<T>::InvalidQuoteSignature);
+				operator_signatures.insert(operator.clone(), *signature);
+			}
+
+			// Ensure all operators have provided a signature
+			for operator in operators.iter() {
+				ensure!(
+					operator_signatures.contains_key(operator),
+					Error::<T>::InvalidQuoteSignature
+				);
+			}
+
+			// Verify each operator's signature
+			// For simplicity, we'll use a basic signature verification
+			for (operator, signature) in operator_signatures.iter() {
+				let operator_preferences = Operators::<T>::get(blueprint_id, operator)?;
+
+				// Convert 65-byte key to 33-byte format for ECDSA verification
+				let mut public_key_bytes = [0u8; 33];
+				if operator_preferences.key.len() >= 33 {
+					// Take the first 33 bytes if the key is larger
+					public_key_bytes.copy_from_slice(&operator_preferences.key[..33]);
+				} else {
+					// This is an error case, but we'll handle it gracefully
+					ensure!(false, Error::<T>::InvalidQuoteSignature);
+				}
+
+				let public_key = ecdsa::Public::from_raw(public_key_bytes);
+
+				// Create a simple message to verify (in production, this would be a hash of the quote details)
+				let message = [0u8; 32];
+
+				// Verify the signature
+				ensure!(
+					sp_io::crypto::ecdsa_verify(signature, &message, &public_key,),
+					Error::<T>::InvalidQuoteSignature
+				);
+			}
```

Comment:
make sure to use the `_prehashed` version of this method, since this one will use blake2 to hash the message hash again.